screen2.0/next.config.js

```diff
@@ -41,6 +41,17 @@ const nextConfig = {
   trailingSlash: false,
   assetPrefix: assetPrefix,
   basePath: basePath,
+  
+  webpack: (config, { isServer }) => {
```

Comment:
genome browser package uses "fs" default file system module of nodejs. Hence I was getting Module not found error. Added below code in next config to resolve this error. Reference (https://stackoverflow.com/questions/64926174/module-not-found-cant-resolve-fs-in-next-js-application)